### ISSUE TITLE
Some prompts and number of iterations result in an entirely black image

### ISSUE THREAD

**Issue by logicReasoner** [bes-dev/stable_diffusion.openvino#14]
For some prompts like "Superman in the mountains" (32 iterations) I get a black image as result. Is that to be expected? The same on an Intel and AMD CPUs.
For other prompts or fewer --num-inference-steps (e.g. 4 which is < 32), it works but with an uglier image.
Any clues?

**Comment by xeruf** [comment]
solution?

**Comment by logicReasoner** [comment]
Nope, but no one had replied so I thought that I was the only one experiencing the above issue. Let me know if you find anything useful on the matter.

**Comment by xeruf** [comment]
Did not experience it, was just curious as I am just now trying it
